src/knoop/Node/src/get.hpp

```diff
@@ -9,3 +9,13 @@ T& get(){
   auto& leaf = std::get< leaf_type >( core );
   return std::get<T>(leaf);
 }
+
+const L& get() const {
```

Comment:
This makes me a little nervous. It seems like I will always return type `L` (the first template parameter.) 
- Can I still get the other parameter types? 
- Is the signature to get the first template parameter type different from getting the second (or any other) template parameter? This should be avoided because it would cause confusion.

Reply:
(1) Yes. (2) It can be, but does not need to be, because both gets still work. The short version allows for (but does not require) shorter/cleaner/less-cluttered notation for one-type Nodes. Which, for instance, we may well use for GNDStk. See more remarks below.